data/strategies/StrategyMakerETHDAIDelegate ETH-C.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyMakerETHDAIDelegate ETH-C",
+  "description": "Locks ETH at MakerDAO via ETH-C to mint DAI. This newly minted DAI is then deposited into the v2 DAI yVault.",
```

Comment:
Do we want to say something about the debt ratio? or may tiny bit longer explanation of this strat?